DESCRIPTION

```diff
@@ -22,7 +25,7 @@ URL: https://insightsengineering.github.io/rlistings/,
     https://github.com/insightsengineering/rlistings/
 BugReports: https://github.com/insightsengineering/rlistings/issues
 Depends:
-    formatters (>= 0.5.10.9004),
+    formatters (>= 0.5.11),
     methods,
     tibble (>= 2.0.0)
```

Comment:
No R dep. Shall we add it? Why is it absent?

Reply:
we can add something here. it's ok if we didn't either